README.md

```diff
@@ -1 +1,113 @@
-Hens never remembers to create a README.
+# gpii-launcher
+
+This package provides a `gpii.launcher` [Fluid component](http://docs.fluidproject.org/infusion/development/UnderstandingInfusionComponents.html)
+that standardizes the launching of fluid components with custom options.  The component is based on [yargs](http://yargs.js.org/) and [Kettle](https://github.com/fluid-project/kettle).
+
+# `gpii.launcher`
+
+The launcher component uses yargs to generate a set of merged options that reflect (in order of precedence):
+
+1. Any options passed on the command line.
```

Comment:
The latest commits supports "pulling" at least environment variables.  Please review and comment.